tests/stokes_solver_fail_gmg/screen-output

```diff
@@ -0,0 +1,72 @@
+
+Vectorization over 2 doubles = 128 bits (SSE2), VECTORIZATION_LEVEL=1
+Number of active cells: 192 (on 4 levels)
+Number of degrees of freedom: 2,724 (1,666+225+833)
+
+*** Timestep 0:  t=0 years, dt=0 years
+   Solving temperature system... 0 iterations.
+   Solving Stokes system... 200+---------------------------------------------------------
+TimerOutput objects finalize timed values printed to the
+screen by communicating over MPI in their destructors.
+Since an exception is currently uncaught, this
+synchronization (and subsequent output) will be skipped
+to avoid a possible deadlock.
+
+
+Exception 'ExcMessage (exception_message.str())' on rank 0 on processing: 
+
+An error occurred in file <utilities.cc> in function
+(line in output replaced by default.sh script)
+The violated condition was: 
+    false
+Additional information: 
+    The iterative Stokes solver in
+    StokesMatrixFreeHandlerImplementation::solve did not converge.
+    
+    The initial residual was: 1.169088e+14
+    The final residual is: 1.375952e-01
+    The required residual for convergence is: 1.169088e-21
+    See output-stokes_solver_fail_gmg/solver_history.txt for the full
+    convergence history.
+    
+    The solver reported the following error:
+    
+    --------------------------------------------------------
+    An error occurred in line <1285> of file
+    </opt/deal.II-v9.3.1/include/deal.II/lac/solver_gmres.h> in function
```

Comment:
Having the path to deal.II here is inconvenient. Do you know why the path is not filtered?